security/pfSense-pkg-suricata/files/usr/local/www/suricata/suricata_rules_edit.php

```diff
@@ -41,14 +41,10 @@
 	exit;
 }
 
-if (!is_array($config['installedpackages']['suricata']['rule'])) {
-	$config['installedpackages']['suricata']['rule'] = array();
-}
-
-$a_rule = &$config['installedpackages']['suricata']['rule'];
+$a_rule = config_get_path("installedpackages/suricata/rule/"{$id}", []);
```

Comment:
PHP syntax error here, the double quote before `{$id}` shouldn't be there

Reply:
Somehow missed running this file through the PHP syntax check test after editing it. Fixed the identified error.